specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -156,6 +156,42 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DataFactory/FactoryNameAvailability": {
```

Comment:
>FactoryNameAvailability [](http://example.com/codeflow?start=69&length=23)

This is a breaking change and needs an approval from the breaking change board. Please start a thread with azbreakchangereview@microsoft.com